### PR TITLE
Added examples to Faucets, fees, fungible tokens and Info API's

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Added examples, and updated description for more clarity to the following API's in the Stacks Blockchain API. 

- faucets
- fees
- fungible tokens
- Info

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -1772,8 +1774,8 @@ paths:
           required: true
           schema:
             type: string
-            enum: [testnet, mainnet]
```
Comment (rafaelcr) [inline docs/openapi.yaml:1775]: I think this enum needs to stay to specify which values are allowed

**Review by rafaelcr — CHANGES_REQUESTED** [review]
Just a small comment but everything looks great otherwise

File: docs/openapi.yaml (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docs/openapi.yaml:1775]: sounds good. Adding it back.

**Comment by LakshmiLavanyaKasturi** [comment]
@rafaelcr: Please review and merge the PR if it LGTY. Thanks!

File: docs/openapi.yaml [hunk 2]
```diff
@@ -2899,17 +2904,19 @@ paths:
       parameters:
         - name: limit
           in: query
-          description: max number of tokens to fetch
+          description: max number of tokens to fetch. limit must be equal to or less than 200.
```
Comment (zone117x) [inline docs/openapi.yaml:2907]: Can this also (or instead) be specified using the `maximum` attribute?

**Comment by LakshmiLavanyaKasturi** [comment]
@zone117x Please review. Thanks!

**Review by zone117x — APPROVED** [review]
LGTM, thanks!

**Comment by LakshmiLavanyaKasturi** [comment]
@rafaelcr: Please merge this PR. Thanks!

**Review by rafaelcr — APPROVED** [review]

**Comment by LakshmiLavanyaKasturi** [comment]
Relates to https://github.com/hirosystems/docs/issues/101

**Pull request merged**